tangos/core/creator.py

```diff
@@ -10,11 +10,11 @@ class Creator(Base):
     __tablename__ = 'creators'
 
     id = Column(Integer, primary_key=True)
-    command_line = Column(String)
+    command_line = Column(String(128))
     dtime = Column(DateTime)
-    host = Column(String)
-    username = Column(String)
-    cwd = Column(String)
+    host = Column(String(128))
+    username = Column(String(128))
+    cwd = Column(String(128))
```

Comment:
This perhaps should also be `Text`

Reply:
Are you suggesting to turn `host` and `username` to `Text` fields as well?